roottest/root/treeformula/event/dt_DrawTest.C

```diff
@@ -128,31 +128,30 @@ TDirectory* GenerateDrawHist(TTree *tree,int level = 3, int quietLevel = 0)
    tree->Draw("fClosestDistance>>hClosestDistance","","goff");
    tree->Draw("fClosestDistance[2]>>hClosestDistance2","","goff");
    tree->Draw("fClosestDistance[9]>>hClosestDistance9","","goff");
-   
+
    // Test variable indexing
    DrawSkippable(tree,"fClosestDistance[fNvertex/2]","hClosestDistanceIndex",
                  (level>0));
    DrawSkippable(tree,"fPx:fPy[fNpoint/6]","fPy[fNpoint/6]>0","hPxInd",(level>0));
 
    // Test on vector of numerical type
-   tree->Draw("fVClosestDistance>>hVClosestDistance","","goff");
-   tree->Draw("fVClosestDistance[2]>>hVClosestDistance2","","goff");
-   tree->Draw("fVClosestDistance[9]>>hVClosestDistance9","","goff");
-   DrawSkippable(tree,"fVClosestDistance[fNvertex/2]","hVClosestDistanceIndex",
-                 (level>0));
+   // tree->Draw("fVClosestDistance>>hVClosestDistance","","goff");
+   // tree->Draw("fVClosestDistance[2]>>hVClosestDistance2","","goff");
+   // tree->Draw("fVClosestDistance[9]>>hVClosestDistance9","","goff");
+   // DrawSkippable(tree,"fVClosestDistance[fNvertex/2]","hVClosestDistanceIndex", (level>0));
```

Comment:
Why do we need to stop testing those?

Reply:
That's a bummer ... we probably need to add them to the generic libEvent then ... they are there to test specific features of the I/O and `TTree::Draw` ....

Reply:
I add vector-based members into basic `root/io/event/Event.h` class and enable drawing test with these memebrs again. I had to rebase PR to be able run CI